app/blog/[title]/page.tsx

```diff
@@ -0,0 +1,35 @@
+import Container from "@/components/body/Container"
+
+export default async function Blog({ params }: { params: { title: string } }) {
+    const formatedTitle = params.title.replaceAll("-", " ")
+    const response = await fetch(process.env.URL + `/api/blogs/getBlogByTitle`, {
```

Comment:
Won't work as the domain will be https://chill31-blog.vercel.app but the actual domain is different as it's in preview